roles/postgres/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+- name: Restart postgres.
+  service:
+    name: postgresql
+    state: "restarted"
+    sleep: 5
```

Comment:
This possibly isn't needed now, as we don't have an init scripts that could cause odd behaviour by exiting immediately on getting a stop signal.

I'm not sure how likely we are to add those, so I think this could be safely removed.